source/gui/__init__.py

```diff
@@ -514,31 +514,31 @@ def __init__(self, frame: MainFrame):
 
 		if not config.isAppX and NVDAState.shouldWriteToDisk():
 			# Translators: The label of a menu item to open the Add-on store
-			item = menu_tools.Append(wx.ID_ANY, _("Add-on &store..."))
+			item = menu_tools.Append(wx.ID_ANY, _("&Add-on store..."))
 			self.Bind(wx.EVT_MENU, frame.onAddonStoreCommand, item)
 
 		if not globalVars.appArgs.secure and not config.isAppX:
 			# Translators: The label for the menu item to open NVDA Python Console.
-			item = menu_tools.Append(wx.ID_ANY, _("Python console"))
+			item = menu_tools.Append(wx.ID_ANY, _("&Python console"))
 			self.Bind(wx.EVT_MENU, frame.onPythonConsoleCommand, item)
 
 		if not globalVars.appArgs.secure and not config.isAppX and not NVDAState.isRunningAsSource():
 			# Translators: The label for the menu item to create a portable copy of NVDA from an installed or another portable version.
-			item = menu_tools.Append(wx.ID_ANY, _("Create portable copy..."))
+			item = menu_tools.Append(wx.ID_ANY, _("&Create portable copy..."))
 			self.Bind(wx.EVT_MENU, frame.onCreatePortableCopyCommand, item)
 			if not config.isInstalledCopy():
 				# Translators: The label for the menu item to install NVDA on the computer.
 				item = menu_tools.Append(wx.ID_ANY, _("&Install NVDA..."))
 				self.Bind(wx.EVT_MENU, frame.onInstallCommand, item)
 			# Translators: The label for the menu item to run the COM registration fix tool 
-			item = menu_tools.Append(wx.ID_ANY, _("Run COM Registration Fixing tool..."))
+			item = menu_tools.Append(wx.ID_ANY, _("&Run COM Registration Fixing tool..."))

```

Comment:
this conflicts with `&Reload plugins`

```suggestion
			item = menu_tools.Append(wx.ID_ANY, _("Run COM Registration &Fixing tool..."))
```

Reply:
> Please consider this carefully. For years now, people such as myself, have been giving, on mailing lists and in other support contexts, the instructions: **Press `NVDA+n`, `t`, `r` ...** to suggest that people attempt certain debugging tasks that include running the CRFT.

Thank you for this valuable feedback.

> Because of add-ons possibly conflicting with it, and `Reload plugins` also, the suggestion to press enter if necessary is often given, at least by me, to complete the instructions.

Pressing Enter is actually necessary, not because of add-ons but because there are two entries beginning with "R" and without explicit shortcut in the Tools menu: "Run COM Registration Fixing tool" and "Reload plugins".

Worse: if an add-on adds an item with explicit "R" shortcut in the "Tools" menu, this will mask both NVDA native items in current NVDA version (2023.2).
 
> Thus having a duplicate key for a minimally used function like `Reload plugins`, doesn't seem like much of a problem.

Personally, I prefer unique shortcut keys when possible since they are more efficient; no need to press enter nor to press various times the letter shortcut.

> However, if you change this key from `r` to `f`, no matter how much sense it makes, you will be invalidating all those support instructions. Or, at the least, calling their validity into question.
> 
> IMO, this key represents one of the top four uses of the tools menu--Add-on Store, Speech Viewer, Create Portable Copy, and the Com Reg Fixing Tool.
> 
> On the other hand, I do not recall anyone ever being told to reload plugins as part of a debugging step (or for any other reason, until very very recently--and in that case the `NVDA+Ctrl+F3` method was given).

I agree with changing the shortcut of "Reload plugins" instead given that there is already `NVDA+control+F3`. Since "p" is already taken by "Python console", "L" could be nice IMO.

Reply:
Once these are fixed it seems like https://github.com/nvaccess/nvda/issues/8913 can be closed too

Reply:
I agree with the conversation and keeping the COM Registration key of R as per @XLTechie - Mostly commenting to note the earlier suggestion to make log viewer L (which would conflict with the suggestion to make reload plugins L)

Reply:
Apologies, I misread the issue